platforms/astbmc/romulus.c

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2017 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <skiboot.h>
+#include <device.h>
+#include <console.h>
+#include <chip.h>
+#include <ipmi.h>
+#include <psi.h>
+#include <npu-regs.h>
+
+#include "astbmc.h"
+
+static bool romulus_probe(void)
+{
+	if (!dt_node_is_compatible(dt_root, "ibm,romulus"))
+		return false;
+
+	/* Lot of common early inits here */
+	astbmc_early_init();
+	psi_set_external_irq_policy(EXTERNAL_IRQ_POLICY_LINUX);
+
+	/* Setup UART for direct use by Linux */
+	uart_set_console_policy(UART_CONSOLE_OS);
```

Comment:
Thanks for the comment and review, we'll send our modified patch to skiboot@lists.ozlabs.org. (trying to close it...)